translations/translations.json

```
@@ -9604,5 +9604,26 @@
     "zh-CN": {
       "noOffersAvailable": "目前没有提供项目"
     }
+  },
+  "zammad": {
```

I think this should be renamed, using `zammad` is just an implementation detail and not really speaking if you don't know what it is.
If we choose to stick to malte specific naming it should probably be 
```suggestion
  "malteHelpForm": {
```